README.md

```diff
@@ -57,24 +50,47 @@ With simple commands, this bot allows you to:
 
 [![Demo Video](https://i9.ytimg.com/vi/aCjcT1CHAzU/mq3.jpg?sqp=COSotI0G&rs=AOn4CLD34jFIlq6GRdmTnK6p3F8O2F-Yig)](https://youtu.be/aCjcT1CHAzU)
 
-## What is new in this version?
+## What's new?
+
+<#TODO: short description here>
 
 Checkout the [this documentation](https://github.com/usmanwardag/dollar_bot/blob/main/docs/Updated_version.pdf)
+
 ## Installation
 
-The below instructions can be followed in order to set-up this bot at your end in a span of few minutes! Let's get started:
+#### Pre-requisite: The Telegram Desktop App
 
-1. Clone this repository to your local system.
+Since DollarBot is built on top of Telegram, you'll first need:
+1. Download the Telegram Desktop Application <a href="https://desktop.telegram.org/">here.</a>
+```https://desktop.telegram.org/```
+2. Create a Telegram account or Sign in.
 
-2. Start a terminal session in the directory where the project has been cloned. Run the following command to install the required dependencies:
-```
-  pip install -r requirements.txt
+Open up your terminal and let's get started:
 
+### MacOS / Ubuntu Users
+
+1. Clone this repository to your local system. 
+```
+   git clone https://github.com/aditikilledar/dollar_bot_SE23/
 ```
-3. Download and install the Telegram desktop application for your system from the following site:
- ```
- https://desktop.telegram.org/
- ```
+2. Start a terminal session in the directory where the project has been cloned. Run the following command and follow the instructions on-screen to complete the installation.
+```
+  bash setup.sh
+```
+
+There, all done!
+
+Our bash scripts for installation are easy to run on MacOS or on UNIX terminals. 
+
+### Windows
+
+With Windows, you'll need to use <a href="https://www.cygwin.com/">Cygwin.</a>
+Cygwin is:
+    a large collection of GNU and Open Source tools which provide functionality similar to a Linux distribution on Windows.
+    a DLL (cygwin1.dll) which provides substantial POSIX API functionality.
+(source: cygwin.com)
+
+Additionally, find more hints on installation <a href="https://stackoverflow.com/questions/6413377/is-there-a-way-to-run-bash-scripts-on-windows">here.</a>
```

Comment:
Shall we not push any software specific steps like this? Maybe we can explain that they need to use some linux based runtime on top of windows? For ex they can use WSL as well right.

Reply:
i.e. Lets just mention that they need to use a unix-like env on top on windows. How they do that can be their choice.

Reply:
Maybe provide examples with their links but not explain in such detail.

Reply:
I agree, but this is just spelling out ONE of the many options for installation. I hear you though, so I'll explicitly mention that it's upto them.